### PR TITLE
[ADD] Accounting: Admonition Block Sunset TaxCloud

### PR DESCRIPTION
**Pull request by tiku-odoo**
Add admonition block about sunsetting TaxCloud in Odoo 17+

THIS IS ONLY TO BE FORWARD PORTED TO saas-16.4 

See here for instructions when merging: https://github.com/odoo/odoo/wiki/Mergebot#example

### CODE REVIEW

**Review by dade-odoo — APPROVED** [review]
Hi again @tiku-odoo , same thing as your other PR for this task: I'll approve on behalf of accounting and let you and Zac handle it :)

File: content/applications/finance/accounting/taxes/taxcloud.rst [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
+   The Odoo TaxCloud integration has begun its decommissioning, starting in
+   Odoo 17. New installations are prohibited in Odoo 17, and in Odoo 18, the TaxCloud module(s)
+   won't exist at all. Odoo recommends the use of the Avatax platform.
+
+   .. seealso::
+      :doc:`avatax`
+
 TaxCloud calculates the sales tax rate in real time for every state, city, and special jurisdiction
 in the United States. It keeps track of which products are exempt from sales tax and in which states
 each exemption applies.
```
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/taxcloud.rst:17]:
```suggestion
TaxCloud calculates the sales tax rate in real-time for every state, city, and special jurisdiction
in the United States. It keeps track of which products are exempt from sales tax, and in which
states each exemption applies.
```

**Review by ksc-odoo — COMMENTED** [review]
hey @tiku-odoo great work! Just a super minor suggestion that you may want to look at, but other than that -- everything looks good. Nicely done! 👍

**Comment by tiku-odoo** [comment]
@samueljlieber This PR is ready for your review when you have a moment. Thanks :+1:

File: content/applications/finance/accounting/taxes/taxcloud.rst [hunk 2]
```diff
@@ -2,9 +2,19 @@
...
+.. warning::
+   ⚠️ Upgrade warning ⚠️
+
+   The Odoo TaxCloud integration has begun its decommissioning, starting in Odoo 17. New
+   installations are prohibited in Odoo 17. In Odoo 18, the TaxCloud module(s) will **not** exist at
+   all. Odoo recommends the use of the Avatax platform, instead.
+
+   .. seealso::
+      :doc:`avatax`
+
```
Comment (samueljlieber) [inline content/applications/finance/accounting/taxes/taxcloud.rst:14]: Pull the seealso admonition out of the warning to be consistent with the previous PR https://github.com/odoo/documentation/pull/5976
```suggestion
.. seealso::
   :doc:`avatax`
```

File: content/applications/finance/accounting/taxes/taxcloud.rst [hunk 3]
```diff
@@ -2,9 +2,19 @@
 TaxCloud integration
 ====================
 
+.. warning::
+   ⚠️ Upgrade warning ⚠️
```
Comment (samueljlieber) [inline content/applications/finance/accounting/taxes/taxcloud.rst:6]: optional - I think this warning is redundant, however, its a stylistic choice so please leave this as is if you disagree with me 🙂
```suggestion
```

**Review by samueljlieber — APPROVED** [review]
Hi @tiku-odoo just a small change needed and a small suggestion- other than that it looks good to me! Approving now, but please make the admonition change for consistency 🙏

File: content/applications/finance/accounting/taxes/taxcloud.rst (hunk 3 above)
Comment (tiku-odoo) [reply content/applications/finance/accounting/taxes/taxcloud.rst:6]: Noted, removed. Thanks

**Comment by tiku-odoo** [comment]
@StraubCreative 

This doc is ready for your review. I've made Sam's edits and he approved it. 

NOTE: This doc is only to be forward ported to saas-16.4 as we covered saas-16.5 in PR: https://github.com/odoo/documentation/pull/5976

**Comment by StraubCreative** [comment]
@fw-bot ignore saas-16.4

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**